services/proxy-service.ts

```diff
@@ -27,6 +28,24 @@ export class ProxyService implements IProxyService {
 		this.$credentialsService.clearCredentials(this.credentialsKey);
 	}
 
+	public async getInfo(): Promise<string> {
```

Comment:
I would expect getInfo to return JSON object with information, not a string. This way we can return it to Fusion for example.